PayPalCheckout/src/main/res/color/paypal_dark_blue.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We were missing some of the Button customization classes (they had been included from Native Checkout) so adding them now that Native Checkout was removed.